src/core/src/preprocess/preprocess_impls.hpp

```diff
@@ -37,7 +37,26 @@ class ModelInfoImpl {
 
 class InputModelInfo::InputModelInfoImpl : public ModelInfoImpl {};
 
-class OutputModelInfo::OutputModelInfoImpl : public ModelInfoImpl {};
+class OutputModelInfo::OutputModelInfoImpl : public ModelInfoImpl {
+public:
+    void set_color_format(const ColorFormat& color_format, const std::vector<std::string>& sub_names = {}) {
+        OPENVINO_ASSERT(color_format == ColorFormat::RGB || color_format == ColorFormat::BGR);
+        m_color_format = color_format;
+        m_planes_sub_names = sub_names;
+        m_color_format_set = true;
+    }
+    bool is_color_format_set() const {
+        return m_color_format_set;
+    }
+    const ColorFormat& get_color_format() const {
+        return m_color_format;
+    }
+
+private:
+    ColorFormat m_color_format = ColorFormat::UNDEFINED;
+    std::vector<std::string> m_planes_sub_names;
```

Comment:
```suggestion
    std::vector<std::string> m_planes_sub_names{};
```
To explicitly init all members.